rules/gsuite_activityevent_rules/gsuite_workspace_calendar_external_sharing.py

```diff
@@ -0,0 +1,24 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    if (
+        not event.get("name", "") != "CHANGE_CALENDAR_SETTING"
```

Comment:
can this just be == instead of double not

Reply:
yeah.. I wasn't crazy about the double negative either. What do you think about the `not all()` form in latest commit?